rest/src/main/scala/org/apache/gearpump/services/RestServices.scala

```diff
@@ -20,59 +20,48 @@ package org.apache.gearpump.services
 
 import akka.actor.{Actor, ActorRef, ActorSystem, Props}
 import akka.io.IO
-import com.gettyimages.spray.swagger._
-import com.wordnik.swagger.model.ApiInfo
+import org.apache.gearpump.cluster.AppMasterToMaster.AppMasterDataDetail
+import org.apache.gearpump.cluster.UserConfig
+import org.apache.gearpump.partitioner.Partitioner
+import org.apache.gearpump.streaming.{AppDescription, TaskDescription, DAG}
+import org.apache.gearpump.util.Graph
 import spray.can._
-import spray.json._
-import spray.routing.HttpService
 
 import scala.concurrent.ExecutionContext
-import scala.reflect.runtime.universe._
 
-class RestServices(masters: ActorRef) extends Actor with HttpService with DefaultJsonProtocol {
-  def actorRefFactory = context
-  implicit val executionContext:ExecutionContext = context.dispatcher
-
-  val appMastersService = new AppMastersService {
-    val master = masters
-    def actorRefFactory = context
-  }
-
-  val appMasterService = new AppMasterService {
-    val master = masters
-    def actorRefFactory = context
-  }
+trait RestServices extends AppMastersService with AppMasterService {
+  val route = appMastersRoute ~ appMasterRoute ~
+    get {
+      pathPrefix("") {
+        pathEndOrSingleSlash {
+          getFromResource("index.html")
```

Comment:
It depends where we point the getFromResource to - it can be from a filesystem. Right now I believe index.html isn't functional since it will actually read from a 'dashboard' index.html in upcoming PR.